e2e/tests/Dashboard/AllElementsPresentNoTrigger.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { test } from '@playwright/test';
+import AggregatesComponent from 'Pages/AggregateComponenet';
+import ChatComponent from 'Pages/ChatComponent';
+import DashboardPage from 'Pages/DashboardPage';
+import DisasterTypeComponent from 'Pages/DisasterTypeComponent';
+import MapComponenet from 'Pages/MapComponent';
+import UserStateComponent from 'Pages/UserStateComponent';
+import { NoTriggerDataSet } from 'testData/testData.enum';
+
+import { FloodsScenario } from '../../../services/API-service/src/scripts/enum/mock-scenario.enum';
+import {
+  getAccessToken,
+  mockFloods,
+  resetDB,
+} from '../../../services/API-service/test/helpers/utility.helper';
+import LoginPage from '../../Pages/LoginPage';
+
+let accessToken: string;
+
+test.beforeEach(async ({ page }) => {
+  // Login
+  const loginPage = new LoginPage(page);
+
+  accessToken = await getAccessToken();
+  await resetDB(accessToken);
```

Comment:
This is fine for now. Let's refactor this in #1600 to use the API instead of the backend function.